src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -66,6 +64,17 @@ else if (element instanceof HaxeVarDeclaration) {
   }
 }
 
+class ImportChecker {
+  public static void check(HaxeImportStatementRegular element, AnnotationHolder holder) {
+    HaxeImportModel importModel = element.getModel();
+    HaxeClassModel aClass = importModel.getHaxeClass();
+    if (aClass == null) {
+      Annotation annotation = holder.createErrorAnnotation(element, "Can't find class");
+      annotation.registerFix(new HaxeRemoveFixer(element));
```

Comment:
It seems like another fix for "add import" would be a more common use case than removing the offending element.   I can see both being useful.